src/streaming/controllers/BufferController.js

```diff
@@ -443,7 +444,33 @@ function BufferController(config) {
         if (!bufferResetInProgress) {
             updateBufferLevel();
             addBufferMetrics();
+            if (needToCatchUp()) {
+                if (!catchingUp) {
+                    catchingUp = true;
+                    eventBus.trigger(Events.PLAYBACK_CATCHUP_START, { sender: instance, bufferLevel: bufferLevel });
+                }
+            } else if (stopCatchingUp()) {
+                catchingUp = false;
+                eventBus.trigger(Events.PLAYBACK_CATCHUP_END, { sender: instance, bufferLevel: bufferLevel });
+            }
+        }
+    }
+
+    function needToCatchUp() {
+        if (playbackController.getIsDynamic() && streamProcessor.getType() === 'video') {
```

Comment:
This implies that only video content will ever be sped up?

Whilst some audio content will be particularly sensitive to playback rate, it should be up to the content provider to determine whether or not to apply speed up to a particular stream type (and the appropriate rate if any).

The usual example of low latency is hearing your neighbours celebrating a goal through the wall. This applies equally to sports audio services.

Reply:
Good catch, I added that check in a first approach, I think it can be removed now